boards/build_scripts/tock_kernel_layout.ld

```diff
@@ -289,11 +289,14 @@ SECTIONS
          * and openocd fails to write it.
          *
          * An issue has been submitted https://github.com/raspberrypi/openocd/issues/25
+         * These bytes broke QEMU. Need to look at this further.
```

Comment:
?

Reply:
Yeah, I did never come back to that. Will revert and when I redo the CHERI virt board I will surely rediscover what went wrong here and revisit making a change.

Reply:
Reverted.